BParkingNano/plugins/ElectronMerger.cc

```diff
@@ -189,15 +210,28 @@ void ElectronMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup con
    edm::Ref<pat::ElectronCollection> ref(pf,ipfele);
    float pf_mva_id = 20.;
    if ( !pf_mvaId_src_Tag_.label().empty() ) { pf_mva_id = float((*pfmvaId)[ref]); }
+   float pf_mva_id_run2 = 20.;
+   if ( !pf_mvaId_src_Tag_run2_.label().empty() ) { pf_mva_id_run2 = float((*pfmvaId_run2)[ref]); }
+   //float pf_mva_id_run3 = 20.;
+   //if ( !pf_mvaId_src_Tag_run3_.label().empty() ) { pf_mva_id_run3 = float((*pfmvaId_run3)[ref]); }
    ele.addUserInt("isPF", 1);
    ele.addUserInt("isLowPt", 0);
-   ele.addUserFloat("ptBiased", 20.);
-   ele.addUserFloat("unBiased", 20.);
-   ele.addUserFloat("mvaId", 20.);
-   ele.addUserFloat("pfmvaId", pf_mva_id);
-   ele.addUserInt("LooseID", ref->electronID("mvaEleID-Fall17-noIso-V1-wpLoose"));
-   ele.addUserInt("MediumID", ref->electronID("mvaEleID-Fall17-noIso-V2-wp90"));
-   ele.addUserInt("TightID", ref->electronID("mvaEleID-Fall17-noIso-V2-wp80"));
+   // Custom IDs
+   ele.addUserFloat("LPEleSeed_Fall17PtBiasedV1RawValue", 20.); // was called "ptBiased"
```

Comment:
A few branches for MVAID are set to 20 by default, is there a reason to keep them in?

Reply:
This is for low pT. When you have an electron recoed only by PF you put that. But in this case we dont have LP at all for now at least

Reply:
Yep, placeholders. Added now just so it's not forgotten later.